src/modules/Microsoft.Resources/deploymentScripts/deploy.bicep

```
@@ -0,0 +1,139 @@
+@description('Required. Display name of the script to be run.')
+param name string
+
+@description('Optional. The ID(s) to assign to the resource.')
+param userAssignedIdentities object = {}
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Type of the script. AzurePowerShell, AzureCLI.')
+@allowed([
+  'AzurePowerShell'
+  'AzureCLI'
+])
+param kind string = 'AzurePowerShell'
+
+@description('Optional. Azure PowerShell module version to be used.')
+param azPowerShellVersion string = '3.0'
```

Should we standardize on 8.0?
```suggestion
param azPowerShellVersion string = '8.0'
```